region/lowernorfair/east.json

```diff
@@ -1464,7 +1513,11 @@
                   {"and": [
                     "heatProof",
                     {"or": [
-                      "canKillRedKiHunters",
+                      {"enemyKill":{
```

Comment:
Did you figure that juking one of them should be pretty doable?

Reply:
Yes the lower one is pretty much a non-issue comming from the door..